src/components/organisms/OcResource/OcResource.spec.js

```diff
@@ -88,4 +93,17 @@ describe("OcResource", () => {
     expect(wrapper.find(".parent-folder").find("a").exists()).toBeFalsy()
     expect(wrapper.find(".parent-folder").attributes("style")).toEqual("cursor: default;")
   })
+
+  it("displays parent folder name default if given", () => {
```

Comment:
Check should be about the default being displayed when the internally built parent folder name is empty. i.e. `displays parent folder name default if calculated name is empty`

Reply:
Suboptimal wording, adopted it to your suggestion